src/views/Info/components/TopTokenMovers/index.tsx

```diff
@@ -73,20 +73,30 @@ const TopTokenMovers: React.FC = () => {
   }, [allTokens])
 
   const increaseRef = useRef<HTMLDivElement>(null)
-  const [increaseSet, setIncreaseSet] = useState(false)
+  const moveLeftRef = useRef<boolean>(true)
+
   // const [pauseAnimation, setPauseAnimation] = useState(false)
```

Comment:
remove commented comment plz, it shouldn't be there

Reply:
yeah, PR overall good, just maybe remove those. My fault for keeping them there though 😬

Reply:
Done, https://github.com/pancakeswap/pancake-frontend/pull/2398